src/routes/tools_routes.js

```diff
@@ -32,6 +33,34 @@ router.use(csrfProtection);
 /* Minimum number of picks for data to show up in Top cards list. */
 const MIN_PICKS = 100;
 
+const chooseIdFromInput = (req) => {
+  //ID is scryfall id or a card name (eg. via Autocomplete hover)
+  const printingPreference = req?.query?.defaultPrinting || req?.user?.defaultPrinting;
+  let { id } = req.params;
+
+  if (!uuidValidate(id)) {
```

Comment:
If we have a uuid then rather than trying to figure a card by name, we can skip that. We end up returning nothing from this method, and then cardFromId gets it